src/App.js

```diff
@@ -20,7 +20,7 @@ function App () {
       <Route path='/inventory'>
         <Main />
       </Route>
-      <Route path='/login'>
```

Comment:
this change is already on another PR

Reply:
Yup. Made a mistake. Hopefully doesn't impact anything...